.github/workflows/test.yml

```
@@ -21,6 +21,7 @@ env:
   GOOGLE_CLIENT_SECRET: ${{ secrets.GOOGLE_CLIENT_SECRET }}
   AWS_ACCESS_KEY: ${{ secrets.AWS_ACCESS_KEY }}
   AWS_SECRET_KEY: ${{ secrets.AWS_SECRET_KEY }}
+  CLOTHES_BUCKET: ${{ secrets.AWS_S3_CLOTHES_BUCKET}
```

actionlint (1.7.7)

got unexpected EOF while lexing end marker }}, expecting '}'

(expression)